rotkehlchen/externalapis/etherscan.py

```diff
@@ -315,6 +315,8 @@ def _query(
                     if status == 0:
                         if result == 'Contract source code not verified':
                             return None
+                        if 'Max daily rate limit reached' in result:
+                            raise RemoteError('Etherscan max daily rate limit reached.')
```

Comment:
How does the normal rate limit reached message look like? Can you find out? As we would want to back off a bit for that. But right now `rate limit reached` is a substring of `Max daily rate limit reached` so if at any point someone moves things around here this will break and make it unreachable.

Reply:
The normal rate limit message is 

```
{"status":"0","message":"NOTOK","result":"Max calls per sec rate limit reached (5/sec)"}
```